spec/mailers/organization_mailer_spec.rb

```diff
@@ -0,0 +1,24 @@
+RSpec.describe OrganizationMailer, type: :mailer do
+  describe "#partner_approval_request" do
+    subject { described_class.partner_approval_request(organization: organization, partner: partner) }
+    let(:partner) { create(:partner) }
+    let(:organization) { partner.organization }
+
+    it "renders the body with correct text with partner information" do
+      expect(subject.body.encoded).to include("<h1> You've received a request to approve the account for #{partner.name}. </h1>")
+      expect(subject.body.encoded).to include("Review This Organization")
+      expect(subject.body.encoded).to include("Review This Organization")
```

Comment:
```suggestion
      expect(subject.body.encoded).to include("Review This Organization")
```

Reply:
Removed the duplicate line - sorry about that!